sorbet/rbi/gems/actionpack.rbi

```diff
@@ -1765,6 +1765,10 @@ class ActionDispatch::RemoteIp::GetIp
   def ips_from(header); end
   def to_s; end
 end
+module ActionController::ApiRendering
+  def render_to_body(options = nil); end
```

Comment:
All the changes in this directory are just in the order of the declarations. Can you remove?

Reply:
Done. Take another look?